test.md

```diff
@@ -0,0 +1 @@
+test
```

Comment:
This file should not appear in your GitHub repo according to the project requirements, as I can see, you did add it correctly to your .gitignore file but I think that you pushed it first before adding it to the gitignore, check this article to solve this issue of removing it from your origin repo to solve this issue please: [Remove file from origin that already exists in .gitignore](https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor)